src/Accounting/index.ts

```diff
@@ -143,6 +143,12 @@ export const calcMinimumMargin: (quote: BigNumber, base: BigNumber, price: BigNu
     }
 };
 
+export const calcMinimumMarginNoExemptions: (base: BigNumber, price: BigNumber, maxLeverage: BigNumber) => BigNumber = (
```

Comment:
Whats the "noExemptions" part of this mean?

Reply:
its just like if you arent leveraged then technically there should be no minMargin, so the other calcMinMargin has a check to see if you are leverged and then returns 0 if you arent. Probably not the best wording